plugins/speculation-rules/settings.php

```diff
@@ -59,13 +59,29 @@ function plsr_get_setting_default(): array {
 	);
 }
 
+/**
+ * Returns the stored setting value for Speculative Loading configuration.
+ *
+ * @since 1.0.0
+ *
+ * @return array{ mode: 'prefetch'|'prerender', eagerness: 'conservative'|'moderate'|'eager' } {
+ *     Stored setting value.
+ *
+ *     @type string $mode      Mode.
+ *     @type string $eagerness Eagerness.
+ * }
+ */
+function plsr_get_stored_setting_value(): array {
+	return plsr_sanitize_setting( get_option( 'plsr_speculation_rules' ) );
```

Comment:
The `plsr_sanitize_setting()` function could perhaps be eliminated in favor of the JSON schema.